java/integTest/src/test/java/glide/TestUtilities.java

```diff
@@ -8,24 +8,33 @@
 
 public class TestUtilities {
 
-    public static BaseTransaction transactionTest(BaseTransaction baseTransaction) {
-        String key1 = "{key}" + UUID.randomUUID();
-        String key2 = "{key}" + UUID.randomUUID();
-        String key3 = "{key}" + UUID.randomUUID();
+    private static final String key1 = "{key}" + UUID.randomUUID();
+    private static final String key2 = "{key}" + UUID.randomUUID();
+    private static final String key3 = "{key}" + UUID.randomUUID();
```

Comment:
key3 is never used, we just jumped to key4. revert those changes, the set command should remain as they were